CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -245,3 +245,31 @@ You can show a bottom navigation bar by passing true to `GiniConfiguration.share
 your own by implementing the `ReviewScreenBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.reviewNavigationBarBottomAdapter`.
 
 You can find more details [here](https://www.figma.com/file/1985HMF83siAXmysSn3dC6/iOS-Gini-Capture-SDK-3.0.0-UI-Customisation?node-id=261%3A8256&t=6sAk7LGf1mi3zV9L-1).
+
+## No results screen
+
+The new no results screen uses the global UI customization options. You can discard the old screen specific
+customizations.
+
+### Breaking Changes
+
+#### Removed localization keys:
+
+`ginicapture.noresults.warning`
+`ginicapture.noresults.collection.header`
+`ginicapture.noresults.gotocamera`
+`ginicapture.noresults.warningHelpMenu`
+
+### New features
+
+#### New localization keys:
+
+`ginicapture.noresult.enterManually`
+`ginicapture.noresult.retakeImages`
+
+#### Bottom navigation bar
+
+You can show a bottom navigation bar by passing true to `GiniConfiguration.shared.bottomNavigationBarEnabled`. There is a default implementation, but you can also use
+your own by implementing the `NoResultBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.noResultNavigationBarBottomAdapter`.
+
```

Comment:
[Request] we need to mention here:

 #### Option to enter details manually

 You can show your own UI for data input if an error occurred and the user clicks the "Enter manually" button on the error screen.